CHANGELOG.md

```diff
@@ -14,6 +14,14 @@
 
 ### STATE BREAKING
 
+## v1.1.1
+
+*Feb 28th, 2025*
+
+### DEPENDENCIES
+
+- Upgrade ibc-go to v7.9.2 to fix ASA-2025-004 [#86](https://github.com/atomone-hub/atomone/pull/86)
```

Comment:
```suggestion
- Upgrade ibc-go to v7.9.2 to fix ASA-2025-004 [#86](https://github.com/atomone-hub/atomone/pull/86) since v7.9.0 has been discontinued
```